Started by timer
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Still waiting to schedule task
Waiting for next available executor on â€˜[8mha:////4C8PgMHPegRpCg8+HpIiIHK02uxvuQktJylkqRHkJagNAAAAox+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgYJ/eT83ILSktQiICOvpCg/Jye1SNfSUh8ABjlyb80AAAA=[0mcontroller-99â€™
Running on [8mha:////4C8PgMHPegRpCg8+HpIiIHK02uxvuQktJylkqRHkJagNAAAAox+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgYJ/eT83ILSktQiICOvpCg/Jye1SNfSUh8ABjlyb80AAAA=[0mcontroller-99 in /root/.jenkins/workspace/test-test-nic-sriov-5times
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (rm project dir)
[8mha:////4EWjgIQS1R18e39T6LdOu3ZpSMSXMburmKwxJHd8e163AAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gwExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jcChmMxwAAAAA==[0m[Pipeline] script
[8mha:////4NOJ/YVWDlnnBpjDdLKlLZkF5LrR387vblsx7fWhZkPlAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0BAkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfpX/cvwAAAAA==[0m[Pipeline] {
[8mha:////4MAm0moKTc8Eo2PcB0LgkHNHI7Xtf5Poej2Mf/l8xh/lAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaoyVLaHxBMgiwpL/WWDdrTyRV/MOEjexcpJJ5k3zXm9Y5wQnSpZ7g73DrB2PYbR18YlSfws0cU9XrgkzBcOlmSR15rygpGJgCWtgJWBjUAfKDm2BrfDqodqg0LaXkup3FNC4boAnMFHNRaXyhTnB7j52mfDn+SuAORZg+9pD/AAeoCAqvAAAAA==[0m[Pipeline] sh
+ ls /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester
1.0.0.0001.version
conf
installation
libs
log_analyzer
README.md
testconf
testLibs
Tests
[8mha:////4F3BY6hez6Yn9lSF4dYULOlz4RwK0Fdsp1OYWtj0vxDtAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKBDglR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYO0G2F+S7bfcBITL9lL0AAAA=[0m[Pipeline] echo
dir is exists and rm dir
[8mha:////4KL1cTFlC/t8/0LRf71hJwxXIUFLIMviWtcIHXR2W8epAAAAox+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6KEzcQ4uTYsfkGFWgvNe9g+hMkv8tf8B4kkTt7pnrOc1xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9YJsL8jLsv8AYAPmjb0AAAA=[0m[Pipeline] sh
+ rm -rf /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester
[8mha:////4GzJynz9IX3CLl34e/1JpYOeLz7FDsqMtIihCtFGqNpZAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRVEiIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPzjQzob0AAAA=[0m[Pipeline] }
[8mha:////4MvApYhNLg1xEFG2iXyHCog7Z9h4MbdAsNSDoK4wJnDgAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRoqBAVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AO8A7Ay9AAAA[0m[Pipeline] // script
[8mha:////4P1g47DHQeYIMdAQPrqVFJW5XTEYSJkMqP0k+p1upCWxAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIREqJBVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AGnsy8m9AAAA[0m[Pipeline] }
[8mha:////4Lc6WrLYEiiAC7rTWY8q4RQPGu1FD+hrgClvEqpzcg2sAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiKMBZVjZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0g35eCrPwAY5Sibb0AAAA=[0m[Pipeline] // stage
[8mha:////4LPzYIRsJlP9vBl87WR2FbvV5590n9jZkqHCRBijPEs5AAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJESDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A6sHc4fBAAAA[0m[Pipeline] stage
[8mha:////4P0yjWfQiEwQxZO7i9obwDXrYwQclNR3i//1CTGbFwBFAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohULDAgpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD1+IXOLCAAAA[0m[Pipeline] { (git clone)
[8mha:////4I3tKNYAz7At397ZkNOs6SFwEn2bzn3ZBhsoegW1dhRwAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKpAQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHNxXfVKy+AAAA[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/intel-innersource/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4CRWXV91kUhqh2bTVXI+2n5U32cQ89nJ7yub5blKQ29bAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKoQQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrt/lO7N9Jjga++AAAA[0m[Pipeline] }
[8mha:////4I4ZVrOFMvh2sVfxnfT9s6+paGHoH9OzkxEwdR4h+nRDAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICFDQoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDQMEcr9Ntfu8AERGGHOvgAAAA==[0m[Pipeline] // stage
[8mha:////4J4BpfnRrLE6odVrNZGgZ11wIRy0HYHBalvcHAcD12j/AAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjEB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INji2eq8EAAAA=[0m[Pipeline] stage
[8mha:////4AgPhGmio4pMNzq5gItmuDxswb3P5dwyIofUIBdMXBTmAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+DOrAycohUdEVMrFEXThCaEJJG/7dJSjpxIq7GHWipxIQHy7ZkvdcbyhjgRMFwp7GzGFvLez+aOfFMobt5ytzRlbeEkbzmjc4NKX1ea0NJw6qCwUbAVmPrKVo0CXbCyYesvERTXVKYt6MAZtUATyjETE4ypG+ZAuzvo4qEP85fAEx9AlYfFquXZ/kBAsTc3cIAAAA=[0m[Pipeline] { (pre-exec-set-args)
[8mha:////4G9dxluK5ut2lxdfo2JVi6OIov1ZH/881VUFzuPzanonAAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggyE0vHypah8QSYIEKY/wkQSeWJvJp3EM2MlVvs7NvmPV+wjgGOFDSzCgeDsTPMu0mXxTKF4eooM0sX1hFGcooJlQX16rSgoKRgSVXDisNGYecoGtQJttzKu2ycRN2cUyjfgUNt+hEeUPFiTjKkL8wBdrepj4Q/z18BzD5B3baf2vs3w5BRkb4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/conf/change_args.py '{"drive_info":{"drive_version":"1.6.4"},"dpdk_dir":"/opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/","vm_image_conf":{"image1":"rhel.img","image2":"rhel2.img","image_dir":"/home/image"},"sut_conf":{"host":"10.112.97.62","pf1":"ens2f0","os":"RHEL"},"pkg_generator":{"host":"10.112.97.60","pf1":"ens2f1"}}'
[8mha:////4FhUQONKPXFSeOjFO92QKbtxv93cUzSuhB7hx3xoWX6KAAAApB+LCAAAAAAAAP9tjbEOwiAURV9rHFwd/QiaNG7GyZWw+AXYIkLJexSodPKL/DX/QbSJk3e4uecu5/mCdQxwpKCZVTgYjJ1h3k26LJYpDFdHmVm6sI4wklNMqCyoV6cFBSUFS6oaVhw2CjtH0aBOsOVW3mXjJOrmnEL5Dhxq04/wgIoXc5IhfWEOsLtNfST8ef4KYPY+Qd3uP9W+Ad3NsHm+AAAA[0m[Pipeline] }
[8mha:////4EpM1Tihe33kOTBucrClpdwMCq1FF6wGwTU/AEOZmYaNAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikKgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrd/lPbN8JP5Su+AAAA[0m[Pipeline] // stage
[8mha:////4JJk4NQMkVX21N+ktknOqKLQw+FD2VU/fvD8dhBPuambAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjIUSDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8AzYdm7bBAAAA[0m[Pipeline] stage
[8mha:////4GToWcz1lEtttcc1F67+obVKf97ChyGWoNBgdNb8ln6iAAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYoABMXWNsnCC0ISQNvq//KS0EyfiatyBlkpMeLBsS9Z7vaGIDCdiJxqLrcdYe9GF3k1JDMTtNdAgGrqImjBSsELZQZGx1VIVJQuLshxWEtYW60DRo0uwkY1+6DJodOU58bQdJeTe3OEJmZzISXP6lpFhe+tNJPxx/gJg7BLku/1sh/lZfAAyQCqkwgAAAA==[0m[Pipeline] { (run nic case)
[8mha:////4M/BBJOrvLliK1a64sS7A0hf4H4seiclg8D1IX9OqKAVAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0YCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDhnK7+9Q+vAGvZp0gvgAAAA==[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics -s -m nicSriov5times --capture=tee-sys --html=/root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-2.1.1
collected 20 items / 19 deselected / 1 selected

npxtester/Tests/testNics/test_network_sriov_stress.py 2022-04-30 02:54:37 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-30/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel2.img-rhel.img]_02_54_37/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel2.img-rhel.img].log, case_logdir: /root/testconfig/testlogs/2022-04-30/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel2.img-rhel.img]_02_54_37
2022-04-30 02:54:37 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel2.img-rhel.img] case ============================
2022-04-30 02:54:37 - test_network_sriov_stress.py - [line:32] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-04-30 02:54:37 - test_network_sriov_stress.py - [line:50] - root - INFO - ssh to the sut1
2022-04-30 02:54:37 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-30 02:54:37 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (publickey) failed.
2022-04-30 02:54:37 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-30 02:54:37 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.112.97.62
2022-04-30 02:54:37 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-30 02:54:38 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (publickey) failed.
2022-04-30 02:54:38 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-30 02:54:38 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.112.97.60
2022-04-30 02:54:38 - test_network_sriov_stress.py - [line:64] - root - INFO - clean up iperf and taskset process...
2022-04-30 02:54:41 - sshInstance.py - [line:50] - root - INFO - pkill -9 iperf
2022-04-30 02:54:41 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 02:54:49 - sshInstance.py - [line:50] - root - INFO - pkill -9 taskset
2022-04-30 02:54:49 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 02:54:57 - sshInstance.py - [line:50] - root - INFO - pkill -9 iperf
2022-04-30 02:54:57 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 02:55:05 - sshInstance.py - [line:50] - root - INFO - pkill -9 taskset
2022-04-30 02:55:05 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 02:55:10 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 3] Opened sftp connection (server version 3)
2022-04-30 02:55:10 - sshInstance.py - [line:201] - root - INFO - start to rmmod irdma....
2022-04-30 02:55:13 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-30 02:55:13 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-30 02:55:18 - sshInstance.py - [line:204] - root - INFO - finished rmmod irdma....
2022-04-30 02:55:18 - sshInstance.py - [line:205] - root - INFO - start to rmmod ice....
2022-04-30 02:55:21 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-30 02:55:25 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 02:55:30 - sshInstance.py - [line:208] - root - INFO - finished rmmod ice....
2022-04-30 02:55:30 - sshInstance.py - [line:210] - root - INFO - start to modprobe ice....
2022-04-30 02:55:33 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-30 02:55:38 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 02:55:43 - sshInstance.py - [line:212] - root - INFO - finished modprobe ice....
2022-04-30 02:55:46 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens2f0
2022-04-30 02:55:46 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 2.50 0x80006a16 1.2852.0
expansion-rom-version: 
bus-info: 0000:4b:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-30 02:55:52 - test_network_sriov_stress.py - [line:71] - root - INFO - try to get ipv6 for ens2f0
2022-04-30 02:55:52 - test_network_sriov_stress.py - [line:73] - root - INFO - try to get ipv6 of pf for redhat...
2022-04-30 02:55:52 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens2f0:
2022-04-30 02:55:58 - sshInstance.py - [line:99] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Sat Apr 30 02:55:47 2022 from 10.239.182.99
[root@WW25ICX2 ~]# nmcli connection delete ens2f0
Connection 'ens2f0' (0604428e-79d0-452a-b8bb-64cec063369f) successfully deleted.
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-04-30 02:56:01 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-04-30 02:56:07 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-04-30 02:56:10 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens2f0:
2022-04-30 02:56:16 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens2f0 ifname ens22f0
Connection 'ens2f0' (c9f15378-412f-471e-898e-0955cf7a613d) successfully added.
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-04-30 02:56:19 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-04-30 02:56:25 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-04-30 02:56:28 - test_network_sriov_stress.py - [line:80] - root - INFO - try to get ipv6 for ens2f1
2022-04-30 02:56:28 - test_network_sriov_stress.py - [line:82] - root - INFO - getting ipv6 in redhat.
2022-04-30 02:56:28 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens2f1:
2022-04-30 02:56:34 - sshInstance.py - [line:99] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Sat Apr 30 02:47:28 2022 from 10.239.182.99
[root@localhost ~]# nmcli connection delete ens2f1
Connection 'ens2f1' (07c658a0-138b-40d5-ae3a-cc5c863a9f66) successfully deleted.
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 02:56:37 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-04-30 02:56:43 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 02:56:46 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens2f1:
2022-04-30 02:56:52 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens2f1 ifname enss2f1
Connection 'ens2f1' (bbfe1645-1cf0-4a5c-8599-de88703a8c33) successfully added.
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 02:56:55 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-04-30 02:57:01 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 02:57:04 - test_network_sriov_stress.py - [line:90] - root - INFO - trying to get ipv6 addr for pf1
2022-04-30 02:57:07 - sshInstance.py - [line:50] - root - INFO - ifconfig ens2f0
2022-04-30 02:57:07 - sshInstance.py - [line:54] - root - INFO - ens2f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::dd05:62c2:9ac6:11d2  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:9c:f7:a4  txqueuelen 1000  (Ethernet)
        RX packets 10  bytes 3037 (2.9 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 45  bytes 7801 (7.6 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-30 02:57:15 - sshInstance.py - [line:50] - root - INFO - ifconfig ens2f1
2022-04-30 02:57:15 - sshInstance.py - [line:54] - root - INFO - ens2f1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::4a2c:61bf:3756:1c38  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a5:9b:45  txqueuelen 1000  (Ethernet)
        RX packets 17932351919  bytes 27147308111765 (24.6 TiB)
        RX errors 0  dropped 322  overruns 0  frame 0
        TX packets 641530773  bytes 46487190199 (43.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-30 02:57:20 - test_network_sriov_stress.py - [line:621] - root - INFO - 
Stress Disable and Enable for 1 time when VF is Assigned VM guest.

2022-04-30 02:57:23 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -s | grep ens2f0 | awk '{print $1;}'
2022-04-30 02:57:23 - sshInstance.py - [line:54] - root - INFO - 0000:4b:00.0

2022-04-30 02:57:28 - test_network_sriov_stress.py - [line:156] - root - INFO - pf_id was found: 0000:4b:00.0

2022-04-30 02:57:28 - test_network_sriov_stress.py - [line:629] - root - INFO - format_pf_id is 0000\:4b\:00.0

2022-04-30 02:57:28 - test_network_sriov_stress.py - [line:632] - root - INFO - pf_id_key: 4b
2022-04-30 02:57:28 - test_network_sriov_stress.py - [line:636] - root - INFO - start to assign ip for ens2f0 of SUT1: 192.168.10.10
2022-04-30 02:57:34 - sshInstance.py - [line:99] - root - INFO - ifconfig ens2f0 192.168.10.10
[root@WW25ICX2 ~]# 
2022-04-30 02:57:37 - test_network_sriov_stress.py - [line:365] - root - INFO - Successfully assigned ip to ethernet: ens2f0
2022-04-30 02:57:37 - test_network_sriov_stress.py - [line:638] - root - INFO - start to assign ip for ens2f1 of SUT2: 192.168.10.11
2022-04-30 02:57:43 - sshInstance.py - [line:99] - root - INFO - ifconfig ens2f1 192.168.10.11
[root@localhost ~]# 
2022-04-30 02:57:46 - test_network_sriov_stress.py - [line:365] - root - INFO - Successfully assigned ip to ethernet: ens2f1
2022-04-30 02:57:46 - test_network_sriov_stress.py - [line:162] - root - INFO - virtulize sut1 pf to 4 vfs
2022-04-30 02:57:49 - sshInstance.py - [line:50] - root - INFO - echo 4 > /sys/class/net/ens2f0/device/sriov_numvfs
2022-04-30 02:57:50 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 02:57:55 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfully Virtualized PF to VFs.
2022-04-30 02:57:58 - sshInstance.py - [line:50] - root - INFO - lspci | grep "Virtual" |grep 4b | awk '{print $1;}'list
2022-04-30 02:57:58 - sshInstance.py - [line:54] - root - INFO - 4b:01.0
4b:01.1
4b:01.2
4b:01.3

2022-04-30 02:58:03 - test_network_sriov_stress.py - [line:185] - root - INFO - Succssfully Gotten vf_id list:

2022-04-30 02:58:06 - sshInstance.py - [line:50] - root - INFO - modprobe vfio-pci

2022-04-30 02:58:06 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 02:58:11 - test_network_sriov_stress.py - [line:205] - root - INFO - start to bind drive in sut1
2022-04-30 02:58:14 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -b vfio-pci 4b:01.0
2022-04-30 02:58:15 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 02:58:20 - test_network_sriov_stress.py - [line:219] - root - INFO - Successfuly bound VF to vfio-pci
2022-04-30 02:58:23 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -b vfio-pci 4b:01.1
2022-04-30 02:58:23 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 02:58:28 - test_network_sriov_stress.py - [line:219] - root - INFO - Successfuly bound VF to vfio-pci
2022-04-30 02:58:28 - test_network_sriov_stress.py - [line:656] - root - INFO - lspci -Dnn | grep 4b:01.1
2022-04-30 02:58:31 - sshInstance.py - [line:50] - root - INFO - lspci -Dnn | grep 4b:01.1
2022-04-30 02:58:31 - sshInstance.py - [line:54] - root - INFO - 0000:4b:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-04-30 02:58:36 - test_network_sriov_stress.py - [line:668] - root - INFO - Successfully get vf device id: 8086:1889
2022-04-30 02:58:36 - test_network_sriov_stress.py - [line:242] - root - INFO - get default network info: virsh net-info default | grep "Active" | awk '{print $2;}'
2022-04-30 02:58:39 - sshInstance.py - [line:50] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-04-30 02:58:40 - sshInstance.py - [line:54] - root - INFO - yes

2022-04-30 02:58:45 - test_network_sriov_stress.py - [line:245] - root - INFO - Default network is active? yes

2022-04-30 02:58:45 - test_network_sriov_stress.py - [line:681] - root - INFO - start to create VMs....
2022-04-30 02:58:45 - test_network_sriov_stress.py - [line:275] - root - INFO - try to create VM with the cmd1: virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/image/rhel2.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0.0,port=5900 --noautoconsole

2022-04-30 02:58:49 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/iimage/rhel2.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.00.0,port=5900 --noautoconsole
usage: virt-install --name NAME --memory MB STORAGE INSTALL [options]
virt-install: error: unrecognized arguments: --osinfo rhel8.2
[root@WW25ICX2 ~]# 
2022-04-30 02:58:52 - test_network_sriov_stress.py - [line:278] - root - INFO - cmd1 is not working, try to create VM with the cmd2: virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/image/rhel2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900 --noautoconsole

2022-04-30 02:58:56 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/iimage/rhel2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900 --nooautoconsole

** (process:46927): [1;33mWARNING[0m **: [34m03:00:18.825[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-04-30 02:58:58 - sshInstance.py - [line:75] - root - INFO - Domain creation completed.
[root@WW25ICX2 ~]# 
2022-04-30 02:59:01 - test_network_sriov_stress.py - [line:292] - root - INFO - Successfully created a VM: VMtest1
2022-04-30 02:59:01 - test_network_sriov_stress.py - [line:275] - root - INFO - try to create VM with the cmd1: virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/image/rhel.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0.0,port=5901 --noautoconsole

2022-04-30 02:59:05 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/iimage/rhel.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0..0,port=5901 --noautoconsole
usage: virt-install --name NAME --memory MB STORAGE INSTALL [options]
virt-install: error: unrecognized arguments: --osinfo rhel8.2
[root@WW25ICX2 ~]# 
2022-04-30 02:59:08 - test_network_sriov_stress.py - [line:278] - root - INFO - cmd1 is not working, try to create VM with the cmd2: virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/image/rhel.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901 --noautoconsole

2022-04-30 02:59:12 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/iimage/rhel.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901 --noaautoconsole

** (process:46983): [1;33mWARNING[0m **: [34m03:00:34.767[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-04-30 02:59:14 - sshInstance.py - [line:75] - root - INFO - Domain creation completed.
[root@WW25ICX2 ~]# 
2022-04-30 02:59:17 - test_network_sriov_stress.py - [line:292] - root - INFO - Successfully created a VM: VMtest2
2022-04-30 02:59:50 - sshInstance.py - [line:50] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-04-30 02:59:50 - sshInstance.py - [line:54] - root - INFO - 192.168.122.172/24

2022-04-30 02:59:55 - test_network_sriov_stress.py - [line:308] - root - INFO - Successfully got VMtest1 IP: 192.168.122.172/24

2022-04-30 02:59:58 - sshInstance.py - [line:50] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-04-30 02:59:58 - sshInstance.py - [line:54] - root - INFO - 192.168.122.58/24

2022-04-30 03:00:03 - test_network_sriov_stress.py - [line:308] - root - INFO - Successfully got VMtest2 IP: 192.168.122.58/24

2022-04-30 03:00:03 - test_network_sriov_stress.py - [line:692] - root - INFO - ip of VM1: 192.168.122.172
2022-04-30 03:00:03 - test_network_sriov_stress.py - [line:693] - root - INFO - ip of VM1: 192.168.122.58
2022-04-30 03:00:03 - test_network_sriov_stress.py - [line:698] - root - INFO - Start ssh VM1 and conf VM1
2022-04-30 03:00:09 - sshInstance.py - [line:99] - root - INFO - ssh-key -R 192.168.122.172
bash: ssh-key: command not found...
[root@WW25ICX2 ~]# 
2022-04-30 03:00:16 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.172
PING 192.168.122.172 (192.168.122.172) 56(84) bytes of data.
64 bytes from 192.168.122.172: icmp_seq=1 ttl=64 time=0.314 ms
2022-04-30 03:00:17 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=2 ttl=64 time=0.228 ms
2022-04-30 03:00:18 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=3 ttl=64 time=2.24 ms
2022-04-30 03:00:19 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=4 ttl=64 time=0.224 ms
2022-04-30 03:00:20 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=5 ttl=64 time=0.272 ms

--- 192.168.122.172 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4091ms
rtt min/avg/max/mdev = 0.224/0.654/2.236/0.791 ms
[root@WW25ICX2 ~]# 
2022-04-30 03:00:29 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.172
root@192.168.122.172's password: 
2022-04-30 03:00:38 - sshInstance.py - [line:99] - root - INFO - 
Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Thu Apr 28 20:58:03 2022 from 192.168.122.1
[root@localhost ~]# 
2022-04-30 03:00:41 - test_network_sriov_stress.py - [line:392] - root - INFO - Successfully ssh VM.
2022-04-30 03:00:47 - sshInstance.py - [line:99] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}}'
0000:00:05.0
[root@localhost ~]# 
2022-04-30 03:00:56 - test_network_sriov_stress.py - [line:414] - root - INFO - successfully got vf pci id of vm: 0000:00:05.0
2022-04-30 03:01:02 - sshInstance.py - [line:99] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
[root@localhost ~]# 
2022-04-30 03:01:07 - test_network_sriov_stress.py - [line:428] - root - INFO - Successfully got vf device name of vm: ens5
2022-04-30 03:01:07 - test_network_sriov_stress.py - [line:440] - root - INFO - try to get ipv6 for ens5
2022-04-30 03:01:07 - test_network_sriov_stress.py - [line:442] - root - INFO - try to get ipv6 in redhat
2022-04-30 03:01:07 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens5:
2022-04-30 03:01:13 - sshInstance.py - [line:99] - root - INFO - nmcli connection delete ens5
Connection 'ens5' (440f45d5-5bb8-49e7-b824-9a6f44bb0bbc) successfully deleted.
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 03:01:16 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-04-30 03:01:22 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 03:01:25 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens5:
2022-04-30 03:01:31 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens5 ifname ens5 [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            5
Connection 'ens5' (4579555d-7c91-43ae-a4bf-574e4b14698a) successfully added.
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 03:01:34 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-04-30 03:01:40 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 03:01:49 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::16f:db0e:2bf9:f395  prefixlen 64  scopeid 0x20<link>
        ether 8a:53:cd:e7:68:3a  txqueuelen 1000  (Ethernet)
        RX packets 22  bytes 5321 (5.1 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 35  bytes 5620 (5.4 KiB)
        TX errors 2  dropped 0 overruns 0  carrier 0  collisions 0
[root@localhost ~]# 
2022-04-30 03:01:52 - test_network_sriov_stress.py - [line:455] - root - INFO - assign ip address for VM...
2022-04-30 03:01:58 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5 192.168.10.12
[root@localhost ~]# 
2022-04-30 03:02:01 - test_network_sriov_stress.py - [line:365] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-04-30 03:02:07 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.172 closed.
[root@WW25ICX2 ~]# 
2022-04-30 03:02:10 - test_network_sriov_stress.py - [line:704] - root - INFO - Start ssh VM2 and conf VM2
2022-04-30 03:02:16 - sshInstance.py - [line:99] - root - INFO - ssh-key -R 192.168.122.58
bash: ssh-key: command not found...
[root@WW25ICX2 ~]# 
2022-04-30 03:02:23 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.58
PING 192.168.122.58 (192.168.122.58) 56(84) bytes of data.
64 bytes from 192.168.122.58: icmp_seq=1 ttl=64 time=0.152 ms
2022-04-30 03:02:24 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.58: icmp_seq=2 ttl=64 time=0.189 ms
2022-04-30 03:02:25 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.58: icmp_seq=3 ttl=64 time=0.227 ms
2022-04-30 03:02:26 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.58: icmp_seq=4 ttl=64 time=6.68 ms
2022-04-30 03:02:27 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.58: icmp_seq=5 ttl=64 time=0.194 ms

--- 192.168.122.58 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4088ms
rtt min/avg/max/mdev = 0.152/1.487/6.675/2.594 ms
[root@WW25ICX2 ~]# 
2022-04-30 03:02:36 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.58
root@192.168.122.58's password: 
2022-04-30 03:02:45 - sshInstance.py - [line:99] - root - INFO - 
Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Thu Apr 28 20:55:57 2022 from 192.168.122.1
[root@localhost ~]# 
2022-04-30 03:02:48 - test_network_sriov_stress.py - [line:392] - root - INFO - Successfully ssh VM.
2022-04-30 03:02:54 - sshInstance.py - [line:99] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}}'
0000:00:05.0
[root@localhost ~]# 
2022-04-30 03:03:03 - test_network_sriov_stress.py - [line:414] - root - INFO - successfully got vf pci id of vm: 0000:00:05.0
2022-04-30 03:03:09 - sshInstance.py - [line:99] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
[root@localhost ~]# 
2022-04-30 03:03:14 - test_network_sriov_stress.py - [line:428] - root - INFO - Successfully got vf device name of vm: ens5
2022-04-30 03:03:14 - test_network_sriov_stress.py - [line:440] - root - INFO - try to get ipv6 for ens5
2022-04-30 03:03:14 - test_network_sriov_stress.py - [line:442] - root - INFO - try to get ipv6 in redhat
2022-04-30 03:03:14 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens5:
2022-04-30 03:03:20 - sshInstance.py - [line:99] - root - INFO - nmcli connection delete ens5
Connection 'ens5' (1464d47b-9072-4665-98d8-d77d42d2e928) successfully deleted.
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 03:03:23 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-04-30 03:03:29 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 03:03:32 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens5:
2022-04-30 03:03:38 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens5 ifname ens5 [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            5
Connection 'ens5' (768b7288-1816-4b89-96ee-28878e09af6c) successfully added.
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 03:03:41 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-04-30 03:03:47 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 03:03:56 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::85a6:bd10:e1df:419d  prefixlen 64  scopeid 0x20<link>
        ether b6:80:ab:8e:24:11  txqueuelen 1000  (Ethernet)
        RX packets 31  bytes 8165 (7.9 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 62  bytes 10950 (10.6 KiB)
        TX errors 1  dropped 0 overruns 0  carrier 0  collisions 0
[root@localhost ~]# 
2022-04-30 03:03:59 - test_network_sriov_stress.py - [line:455] - root - INFO - assign ip address for VM...
2022-04-30 03:04:05 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5 192.168.10.13
[root@localhost ~]# 
2022-04-30 03:04:08 - test_network_sriov_stress.py - [line:365] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-04-30 03:04:14 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.58 closed.
[root@WW25ICX2 ~]# 
2022-04-30 03:04:17 - test_network_sriov_stress.py - [line:712] - root - INFO - Ping two vms IP from SUT:
2022-04-30 03:04:21 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.196 ms
2022-04-30 03:04:22 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.122 ms
2022-04-30 03:04:23 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.120 ms
2022-04-30 03:04:24 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.123 ms
2022-04-30 03:04:25 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.122 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4113ms
rtt min/avg/max/mdev = 0.120/0.136/0.196/0.032 ms
[root@WW25ICX2 ~]# 
2022-04-30 03:04:32 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.158 ms
2022-04-30 03:04:33 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.121 ms
2022-04-30 03:04:34 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.125 ms
2022-04-30 03:04:35 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.129 ms
2022-04-30 03:04:36 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.123 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4107ms
rtt min/avg/max/mdev = 0.121/0.131/0.158/0.015 ms
[root@WW25ICX2 ~]# 
2022-04-30 03:04:39 - test_network_sriov_stress.py - [line:720] - root - INFO - ping VMtest2 IP from VMtest1
2022-04-30 03:04:45 - sshInstance.py - [line:99] - root - INFO - ssh-key -R 192.168.122.172
bash: ssh-key: command not found...
[root@WW25ICX2 ~]# 
2022-04-30 03:04:52 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.172
PING 192.168.122.172 (192.168.122.172) 56(84) bytes of data.
64 bytes from 192.168.122.172: icmp_seq=1 ttl=64 time=0.339 ms
2022-04-30 03:04:53 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=2 ttl=64 time=0.215 ms
2022-04-30 03:04:54 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=3 ttl=64 time=0.154 ms
2022-04-30 03:04:55 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=4 ttl=64 time=0.210 ms
2022-04-30 03:04:56 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=5 ttl=64 time=0.231 ms

--- 192.168.122.172 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4114ms
rtt min/avg/max/mdev = 0.154/0.229/0.339/0.063 ms
[root@WW25ICX2 ~]# 
2022-04-30 03:05:05 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.172
root@192.168.122.172's password: 
2022-04-30 03:05:14 - sshInstance.py - [line:99] - root - INFO - 
Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Fri Apr 29 15:01:57 2022 from 192.168.122.1
[root@localhost ~]# 
2022-04-30 03:05:17 - test_network_sriov_stress.py - [line:392] - root - INFO - Successfully ssh VM.
2022-04-30 03:05:21 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.235 ms
2022-04-30 03:05:22 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.031 ms
2022-04-30 03:05:23 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.026 ms
2022-04-30 03:05:24 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.024 ms
2022-04-30 03:05:25 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.026 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 113ms
rtt min/avg/max/mdev = 0.024/0.068/0.235/0.083 ms
[root@localhost ~]# 
2022-04-30 03:05:34 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.172 closed.
[root@WW25ICX2 ~]# 
2022-04-30 03:05:37 - test_network_sriov_stress.py - [line:728] - root - INFO - ping VMtest2 IP from VMtest1
2022-04-30 03:05:41 - sshInstance.py - [line:75] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.453 ms
2022-04-30 03:05:42 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.196 ms
2022-04-30 03:05:43 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.180 ms
2022-04-30 03:05:44 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.201 ms
2022-04-30 03:05:45 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.231 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4081ms
rtt min/avg/max/mdev = 0.180/0.252/0.453/0.102 ms
[root@WW25ICX2 ~]# 
2022-04-30 03:05:51 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens2f0/device/sriov_numvfs
2022-04-30 03:05:54 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 03:05:59 - test_network_sriov_stress.py - [line:327] - root - INFO - Successfully removed VFs from PF.
2022-04-30 03:06:02 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest1
2022-04-30 03:06:02 - sshInstance.py - [line:54] - root - INFO - Domain VMtest1 destroyed


2022-04-30 03:06:07 - test_network_sriov_stress.py - [line:340] - root - INFO - Successfully shut down the VM: VMtest1
2022-04-30 03:06:10 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest1
2022-04-30 03:06:10 - sshInstance.py - [line:54] - root - INFO - Domain VMtest1 has been undefined


2022-04-30 03:06:15 - test_network_sriov_stress.py - [line:350] - root - INFO - Successfully removed the VM: VMtest1
2022-04-30 03:06:18 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest2
2022-04-30 03:06:18 - sshInstance.py - [line:54] - root - INFO - Domain VMtest2 destroyed


2022-04-30 03:06:23 - test_network_sriov_stress.py - [line:340] - root - INFO - Successfully shut down the VM: VMtest2
2022-04-30 03:06:26 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest2
2022-04-30 03:06:26 - sshInstance.py - [line:54] - root - INFO - Domain VMtest2 has been undefined


2022-04-30 03:06:31 - test_network_sriov_stress.py - [line:350] - root - INFO - Successfully removed the VM: VMtest2
.2022-04-30 03:06:31 - test_network_sriov_stress.py - [line:102] - root - INFO - removing VMs: VMtest1, VMtest2
2022-04-30 03:06:34 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest2
2022-04-30 03:06:34 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest2'


2022-04-30 03:06:39 - test_network_sriov_stress.py - [line:340] - root - INFO - Successfully shut down the VM: VMtest2
2022-04-30 03:06:42 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest2
2022-04-30 03:06:43 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest2'


2022-04-30 03:06:48 - test_network_sriov_stress.py - [line:353] - root - ERROR - Failed to removed the VM VMtest2
2022-04-30 03:06:51 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest1
2022-04-30 03:06:51 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest1'


2022-04-30 03:06:56 - test_network_sriov_stress.py - [line:340] - root - INFO - Successfully shut down the VM: VMtest1
2022-04-30 03:06:59 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest1
2022-04-30 03:06:59 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest1'


2022-04-30 03:07:04 - test_network_sriov_stress.py - [line:353] - root - ERROR - Failed to removed the VM VMtest1
2022-04-30 03:07:07 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens2f0/device/sriov_numvfs
2022-04-30 03:07:07 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 03:07:12 - test_network_sriov_stress.py - [line:107] - root - INFO - try to reset network for ens2f0
2022-04-30 03:07:12 - test_network_sriov_stress.py - [line:109] - root - INFO - try to reset network for redhat...
2022-04-30 03:07:12 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens2f0:
2022-04-30 03:07:18 - sshInstance.py - [line:99] - root - INFO - nmcli connection delete ens2f0
Connection 'ens2f0' (c9f15378-412f-471e-898e-0955cf7a613d) successfully deleted.
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-04-30 03:07:21 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-04-30 03:07:27 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-04-30 03:07:30 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens2f0:
2022-04-30 03:07:36 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens2f0 ifname ens22f0
Connection 'ens2f0' (278ca0bc-a881-4f1d-b0c4-1a8170859854) successfully added.
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-04-30 03:07:39 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-04-30 03:07:45 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-04-30 03:07:48 - test_network_sriov_stress.py - [line:116] - root - INFO - try to reset network for ens2f1
2022-04-30 03:07:48 - test_network_sriov_stress.py - [line:118] - root - INFO - try to reset network for redhat
2022-04-30 03:07:48 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens2f1:
2022-04-30 03:07:54 - sshInstance.py - [line:99] - root - INFO - nmcli connection delete ens2f1
Connection 'ens2f1' (bbfe1645-1cf0-4a5c-8599-de88703a8c33) successfully deleted.
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 03:07:57 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-04-30 03:08:03 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 03:08:06 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens2f1:
2022-04-30 03:08:12 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens2f1 ifname enss2f1
Connection 'ens2f1' (f7c7d4f6-242d-4440-b674-c56a7203fbdc) successfully added.
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 03:08:15 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-04-30 03:08:21 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-04-30 03:08:27 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-30 03:08:27 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 03:08:32 - test_network_sriov_stress.py - [line:126] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-30 03:08:32 - test_network_sriov_stress.py - [line:128] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-30/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel2.img-rhel.img]_02_54_37
2022-04-30 03:08:35 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-30 03:08:35 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 03:08:40 - test_network_sriov_stress.py - [line:130] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-30 03:08:43 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/*.txt
2022-04-30 03:08:43 - sshInstance.py - [line:54] - root - INFO - 
2022-04-30 03:08:48 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-30 03:08:48 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel2.img-rhel.img] case ============================
                  [100%]

=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:116
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:116: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:169
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:169: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:102
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:102: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:141
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:141: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:617
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:617: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:744
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:744: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:984
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:984: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_network_sriov_stress.py:1123
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:1123: PytestUnknownMarkWarning: Unknown pytest.mark.nicIperfTcpSUTGuestStress - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicIperfTcpSUTGuestStress

npxtester/Tests/testNics/test_network_sriov_stress.py:1124
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:1124: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:119
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:119: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:156
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:156: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:157
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:157: PytestUnknownMarkWarning: Unknown pytest.mark.connect - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.connect

npxtester/Tests/testNics/test_nics.py:189
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:189: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:190
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:190: PytestUnknownMarkWarning: Unknown pytest.mark.static_ip - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.static_ip

npxtester/Tests/testNics/test_nics.py:221
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:221: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:222
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:222: PytestUnknownMarkWarning: Unknown pytest.mark.test - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.test

npxtester/Tests/testNics/test_nics.py:308
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:308: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:334
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:334: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:377
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:377: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html -
========== 1 passed, 19 deselected, 19 warnings in 867.46s (0:14:27) ===========
[8mha:////4LeiMYyvaTAszhIQ10pdIeDepJ20hGjpoNTSDQ54vTlOAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0SCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQGMKkS1vgAAAA==[0m[Pipeline] }
[8mha:////4I/tRoD/KV6lJ76TTEJ8vusLD5mbZiDJ1z7tZBIwhHThAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKCQKJBqWgtN7zAJMY4se6C7eBUvIiv8QcsIlGxxWpnm3m9YRk81OwNdpp6S6GxOLjR5IWJfX91nLDjCzZMgZ1GqZPkVp9mlBw1zClKWAhYaWocB0smwlp06qEqp8hU5+jzdxRQ2vYOTyhENkfl4xcmD5vb2Aamn+evAKZhiFDut7l2hw9mQDtDvgAAAA==[0m[Pipeline] // stage
[8mha:////4CAP7WQdC2FMpNczqgZSKdPjM+Zu6Lz59I8VEE3/AlUOAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjBB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INOibpDcEAAAA=[0m[Pipeline] stage
[8mha:////4Db8sOlSRxZnVxb7xJBDSAteoF0PgC9lkQldzKIJFwYiAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUwIiYukZZOEFoQkgb/V9+UtqJE3E17kBLJSY8WLYl673eUESGE7ETjcXWY6y96ELvpiQG4vYaaBANXURNGClYoeygyNhqqYqShUVZDisJa4t1oOjRJdjIRj90GTS68px42o4Scm/u8IRMTuSkOX3LyLC99SYS/jh/ATB2CfLDbrb9/Cw+2YMO4MIAAAA=[0m[Pipeline] { (collect logs)
[8mha:////4LN/0LMxu1hyybTBeE6CHwDx82bv/lkvxFk3cdgHTSuWAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikRJR0VppeIFJjLFj3Tm2g1PxIr7GHwhEomKL1c4283zBOgY4ctBoFfWGYmvQu1HPCzOH/uo4o+ULtkyRncJG5YY7dVqw4aRgSVHCSsBGUes4GtIJtsLKu6ycJF2dU5i/g4DSdAM8oBCzOcmQvjAF2N3GLjL9PH8FMPkEZb3/VO3favo5XL4AAAA=[0m[Pipeline] sh
+ sed -i $'s/\r//g' /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4FgvIflgycWoJq7MsZycbV9d7HENRG2WSX14oF+Me/GuAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiAR2iSmu54QUmNsaJdWdsh6TiRXyNPxCIRMUWq51t5vmCZYpw5GixNdQ5So3D4Hs7LRw4dhfPA7Z8xoYpsTcozSBZm3pGydnAnKKEhYCVocZzcmQzrEWr7qryimx1ynH6DgJKp2/wgEJM5qxi/sIYYXPtdWL6ef4KYAwZyt32U/vwBq1seBO+AAAA[0m[Pipeline] sh
+ chmod a+x /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4P37N80+NRbzyFlga2j1WDon6H/4VmnQwrRysqQpQX/3AAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigagQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUO62n9qHN+xdYwq+AAAA[0m[Pipeline] sh
+ sh /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4P3ih/vRXCzaXCkS1CQDBQXrb3I4EZcCr0NGGH+tKlR4AAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigUSDqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFDu9p/avgEjVhSLvgAAAA==[0m[Pipeline] }
[8mha:////4ISV0Ptmba0km8a80Gq+UClogdZnPDxCJXroOoeI5sULAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiREGBqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFDu9p/avgE81EHZvgAAAA==[0m[Pipeline] // stage
[8mha:////4LmA5LOJ1yGOO5YQ6RYn+/szIsYEkfxAArsUKSFs0Xa0AAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIpoEBCVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5Nu9INt9AE/B/a+9AAAA[0m[Pipeline] }
[8mha:////4KVKO0UZMawo8C9tjXN4L1+ryWbU4zmvD2sDlsKoyYtXAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiIUVI5jZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0gL/eCrPwAXH35DL0AAAA=[0m[Pipeline] // node
[8mha:////4A8vvhfR6oegt+2XAbvn3Z0Fx6j3lb1QUh2abilCs+OGAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMJBiA5R0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77eCbPcBAx/ra70AAAA=[0m[Pipeline] End of Pipeline
Finished: SUCCESS
